bakery/src/scripts/assemble_book_group.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+exec 2> >(tee "${ASSEMBLED_OUTPUT}/stderr" >&2)
+
+shopt -s globstar nullglob
+for collection in "${RAW_COLLECTION_DIR}/collections/"*; do
+#for collection in $(find "${RAW_COLLECTION_DIR}/collections/" -type f); do
+    slug_name=$(basename "$collection" | awk -F'[.]' '{ print $1; }')
+    rm -rf temp-assembly
```

Comment:
maybe not related to this PR itself, but I would guess this line is not needed. maybe something leftover from development?
```suggestion
```